src/Capability/Discovery/SchemaValidator.php

```diff
@@ -220,7 +220,7 @@ private function formatValidationError(ValidationError $error): string
                 break;
             case 'type':
                 $expected = implode('|', (array) ($args['expected'] ?? []));
-                $used = $args['used'] ?? 'unknown';
```

Comment:
Looking at the `opis/json-schema` package, I cannot find that `$args['used']` is ever set.